contrib/openapi.yml

```diff
@@ -199,3 +243,41 @@ definitions:
         type: string
         description: |
           A unique identifier representing assigned OpenStack environment.
```

Comment:
But probably it would be better to rename it in a separate PR since it isn't related to the issue https://github.com/sc68cal/Kostyor/issues/34. I'll create another PR for renaming "OpenStack environment" into "cluster".